src/main/java/ava/AVA.java

```diff
@@ -0,0 +1,119 @@
+package ava;
+
+import ava.task.Task;
+import ava.task.TaskManager;
+
+import java.io.PrintStream;
+import java.util.List;
+
+public class AVA {
+
+    /**
+     * Current user input being processed by AVA.
+     */
+    private String currentInput;
+
+    /**
+     * TaskManager for AVA
+     * <br>
+     * initialized on ava's creation
+     * <br><br>
+     * <code>final</code> to avoid unnecessary modifications
+     */
+    private final TaskManager taskManager;
+
+    /**
+     * default constructor for AVA
+     */
+    public AVA() {
+        taskManager = new TaskManager();
+    }
+
+    /**
+     * Decides if AVA is running or not
+     * //TODO: switch to a state based system like Operating System Threads
```

Comment:
Is there a reason why you put the TODO in the javadocs? Is there a reason why the TODO format is not consistent with the next TODO?